### DISCUSSION TITLE
s/google-adsense-payment-china-merchants-bank

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#68] [Announcements]
# s/google-adsense-payment-china-merchants-bank

聊聊我是如何赚到美金，以及怎样使用国内银行卡进行收款的

https://blog.ops-coffee.cn/s/google-adsense-payment-china-merchants-bank

**Comment by kxingking** [comment]
看到这里，关掉了adblock :)

**Reply by ops-coffee** [reply]
😄

**Comment by note4** [comment]
其实谷歌的广告还是可以接受的。
